### PR TITLE
SDK crate

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: sdk/Cargo.toml
Comment (Thoralf-M) [inline sdk/Cargo.toml:78]: Why `"fern-logger"` here?

Comment (thibault-martinez) [reply sdk/Cargo.toml:78]: It's only used for the `init_logger` function of the message interface

Comment (Thoralf-M) [reply sdk/Cargo.toml:78]: Ah Brord added the function also to main crate, I thought it was only in the bindings

File: .github/workflows/bindings-nodejs.yml
Comment (Thoralf-M) [inline .github/workflows/bindings-nodejs.yml:14]: What about the wallet bindings?
Exclude should be like this 
```
      - "!/bindings/**" # Exclude all bindings
```
and include not sure, maybe we wait until we merged it

Comment (thibault-martinez) [reply .github/workflows/bindings-nodejs.yml:14]: Yeah tbh I think I'd merge without taking care too much of the CI for now, it needs a lot of changes because of the merge.

Comment (thibault-martinez) [reply .github/workflows/bindings-nodejs.yml:14]: I've done the bare minimum so we could merge and rebase other PRs ASAP, I'll properly fix them all later

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**
